src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -35,27 +35,40 @@ interface EmailPreviewProps {
   onUpdate: () => void
 }
 
-const fontStyling = '<style>*{font-family:Avenir Next;}</style>'
+const styling = '<style>*{font-family:"Avenir Next";}</style>'
+const fontSrc =
+  '<link rel="stylesheet" href="https://static.amsterdam.nl/fonts/fonts.css"/>'
 
 const EmailPreview: FC<EmailPreviewProps> = ({
   emailBody,
   onUpdate,
   onClose,
-}) => (
-  <ModalContainer>
-    <ModalHeader title="Controleer bericht aan melder" onClose={onClose} />
-    <StyledIframe
-      data-testid="emailBodyIframe"
-      srcDoc={emailBody.concat(fontStyling)}
-      height="500"
-    />
-    <StyledFormFooter
-      cancelBtnLabel="Wijzig"
-      onCancel={onClose}
-      submitBtnLabel="Verstuur"
-      onSubmitForm={onUpdate}
-    />
-  </ModalContainer>
-)
+}) => {
+  const htmlParts = emailBody.split(/(<head>)/)
+  const styledHtml = [
+    ...htmlParts[0],
+    ...htmlParts[1],
+    ...fontSrc,
+    ...styling,
+    ...htmlParts[2],
+  ].join('')
+
+  return (
+    <ModalContainer>
+      <ModalHeader title="Controleer bericht aan melder" onClose={onClose} />
+      <StyledIframe
+        data-testid="emailBodyIframe"
+        srcDoc={styledHtml}
+        height="500"
```

Comment:
The iframe's CSS has both a width and a height of 100%. Does this prop (`height`) serve any purpose?

Reply:
not any more. it is removed